home/management/commands/broken_links_clean_up.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+from wagtail.models import Page
+
+from home.models import ContentPage, OrderedContentSet
+
+
+class Command(BaseCommand):
+    help = "Clean broken links for pages"
+
+    def handle(self, *args, **options):
+
+        # Only query live pages
+        content_pages = ContentPage.objects.live()
```

Comment:
I think we want to query all pages, not just live pages. Otherwise we could have a page in draft with this issue, it goes unnoticed because this script doesn't pick it up, and then we push it live we have a broken live page.